tests/conftest.py

```diff
@@ -1,11 +1,31 @@
 import asyncio
+import uuid
+from distutils.version import StrictVersion
+from os import environ as ENV
 
 import pytest
 
 from aiodocker.docker import Docker
 from aiodocker.exceptions import DockerError
 
 
+_api_versions = {
```

Comment:
Indeed. But the docker doc about the API is....well...hard.
And there is not really an official way to implement the API (I was cross-checking the docker-py code....) and you are right it's not our job but this library has to work with Docker, otherwise it's useless no matter how well is implemented.